src/utils/ejson.ts

```diff
@@ -0,0 +1,46 @@
+import { EJSON } from 'bson';
+import type { Document } from 'bson';
+
+const isObject = (value: unknown) =>
+  value !== null && typeof value === 'object';
+
+function simplifyEJSON(documents: Document[] | Document): Document {
```

Comment:
also nit: since it is either one or many documents, we could call the argument `item` or something, but `documents` confuses when reading code, because it can be actually one document that is not an array.

Reply:
yeah, I wasn't sure about the naming on this one. I can use item if that's more clear.